Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -2832,7 +2832,7 @@ MediaPlayer::SupportsType MediaPlayerPrivateGStreamer::supportsType(const MediaE
     result = gstRegistryScanner.isContentTypeSupported(GStreamerRegistryScanner::Configuration::Decoding, parameters.type, parameters.contentTypesRequiringHardwareSupport);
 
 #if PLATFORM(BROADCOM)
-    if (String::fromLatin1("video/webm") == parameters.type.containerType()) {
+    if (String::fromLatin1("video/webm") == parameters.type.containerType() && parameters.type.codecs().isEmpty()) {
```

Comment:
1. in this case we are accepting combination of webm+vp8?
2. we want to have similar logic for MSE path. Now i's the same?

Reply:
1. in this case we're just asked about "video/webm", without the list of codecs. if we return "maybe" (as we do now), we'll just get webm + vp8 content, without further queries.
2. no, it's not - for MSE path, MediaSource.isTypeSupported("video/webm") returns true, while with vp8 codec - 'video/webm; codecs="vp8"' - it returns false. it's the same currently on 2.22 though